cases/forms/case.py

```diff
@@ -120,8 +121,14 @@ class Meta:
 
     type = forms.ChoiceField(widget=forms.RadioSelect, required=True)
     notes = action_notes_field()
+    files = forms.FileField(
+        label="Attachments",
+        widget=forms.ClearableFileInput(attrs={"multiple": True}),
+        required=False,
+    )
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args, case=None, **kwargs):
+        self.case = case
```

Comment:
Needed the case before `save()` for checking the remaining space when validating uploaded files.